### DISCUSSION TITLE
New input type suggestion

### DISCUSSION THREAD

**Discussion by Hael616** [saadeghi/daisyui#1678] [Ideas / feature requests]
I'm suggesting a new input type variant I don't know if you'll be okay with it, but I'll attach two image files below to show what I'm talking about.

### What the input should look like before focus event

![Screenshot (77)](https://user-images.githubusercontent.com/71190585/224027861-8c81dc33-cc9d-4536-88c9-01b2e8ff18e3.png)

### What it should look like while it's focused

![Screenshot (78)](https://user-images.githubusercontent.com/71190585/224028781-043aefd4-8aa3-4db0-8909-3c32a3da4a17.png)

Notice how the label of each input moves towards the edge of their border-top when clicked(Don't know how to word this properly), I think it'll be good to have varieties of inputs 

**I think it's worth noting that I'm just starting out with Daisy UI but I'll gladly help out with implementing this suggestion.**

**Comment by saadeghi** [comment]
It's a Material Design pattern. I don't really want to copy this design pattern from Material because it doesn't really fix any problem or add any value. It's just a design decision.  

However, you can add it to your project like this:  
- https://play.tailwindcss.com/ntvQhr1ABY
- or: https://play.tailwindcss.com/8mIaMMvGRY?file=css

**Reply by Hael616** [reply]
Oh okay no problem

Thanks for the suggestion by the way, I really appreciate.
